nima/http2/webserver/src/test/resources/application.yaml

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2022 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+server:
+  port: 8079
+  host: 127.0.0.1
+
+  sockets:
+    - name: http
+
+  connection-providers:
+    http_2:
+      max-frame-size: 8192
+      max-header-size: 4096
```

Comment:
Is the header list size really size of headers in bytes?

Reply:
Javadoc in Helidon says nothing about units of those values.
Defaults in Http2Setting are:
- MAX_FRAME_SIZE: 16_384L
- MAX_HEADER_LIST_SIZE: Http2SettingBase.MAX_UNSIGNED_INT

[RFC 7540](https://www.rfc-editor.org/rfc/rfc9113#SETTINGS_MAX_FRAME_SIZE) defines the sizes as "units of octets".